rebar.config

```diff
@@ -205,18 +205,19 @@
                                        "_build/dev1/lib/aeutils/priv/aeternity_config_schema.json",
                                        "{{output_dir}}/data/aeternity_config_schema.json"}]}
                            ]},
-                     {erl_opts, [{d, 'TEST'}]}
+                     {erl_opts, [{d, 'TEST'}, {d, 'DEBUG_INFO'}]}
                     ]
             },
             {test, [{relx, [{dev_mode, true},
                             {include_erts, false},
                             {include_src, true},
                             {sys_config, "./config/dev1/sys.config"},
                             {vm_args, "./config/dev1/vm.args"}]},
+                    {erl_opts, [{d, 'DEBUG_INFO'}]},
                     {dist_node, [{setcookie, 'aeternity_cookie'},
                                  {sname, 'aeternity_ct@localhost'}]},
                     {deps, [{meck, "0.8.12"},
-                            {aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"4ae2472"}}},
+                            {aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"122a6e0"}}},
```

Comment:
You need to have a Ceres compiler...

Reply:
The ceres branch should be rebased on top of master after merging the debugger PR in the aesophia repository, only after that we can have a ceres compiler here.

Alternatively, the ceres branch in aesophia could be merged to master, and then the debugger branch could be rebased on top of the new master.

I think the first option is more likely to be the right one. Could you suggest another solution for this?

Reply:
The ceres branch of aesophia should be kept separate until we get nearer to the hard fork, so first option is the right one. In the meantime you could rebase the ceres branch (new_ceres?)  on top of the debugger PR in aesophia to make sure it works with this PR 😅